modules/passkey/contracts/SafeWebAuthnSignerFactory.sol

```diff
@@ -32,7 +32,8 @@ contract SafeWebAuthnSignerFactory is ISafeSignerFactory {
     /**
      * @inheritdoc ISafeSignerFactory
      */
-    function getSigner(uint256 x, uint256 y, P256.Verifiers verifiers) public view override returns (address signer) {
```

Comment:
I don't think its a good idea to edit safe source code.
Why wont we just add a munged file to replace the whole file with the harness addition?

Reply:
Removed changes from source code and move to munged